partitioner/src/main/java/io/confluent/connect/storage/partitioner/DailyPartitioner.java

```diff
@@ -16,42 +16,20 @@
 
 package io.confluent.connect.storage.partitioner;
 
-import org.apache.kafka.common.config.ConfigException;
-import org.joda.time.DateTimeZone;
-
-import java.util.Locale;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
-import io.confluent.connect.storage.common.StorageCommonConfig;
-
 public class DailyPartitioner<T> extends TimeBasedPartitioner<T> {
   @Override
   public void configure(Map<String, Object> config) {
-    String delim = (String) config.get(StorageCommonConfig.DIRECTORY_DELIM_CONFIG);
-    String pathFormat = "'year'=YYYY" + delim + "'month'=MM" + delim + "'day'=dd";
+    long partitionDurationMs = TimeUnit.DAYS.toMillis(1);
 
-    String localeString = (String) config.get(PartitionerConfig.LOCALE_CONFIG);
```

Comment:
Did a bit refactoring on these. `super.configure()` is already pulling most of these